From f16dbbd6d1d3c407beafdf49e01fcd7744c51111 Mon Sep 17 00:00:00 2001
From: Hongxu Jia <hongxu.jia@windriver.com>
Date: Sat, 6 Jul 2019 11:05:11 +0800
Subject: [PATCH] livepayload: fix mount install tree failed

...
  File "/usr/lib64/python3.7/site-packages/pyanaconda/payload/livepayload.py", line 85, in setup
    raise PayloadInstallError("Failed to mount the install tree")
...

Try mounting with noload

Upstream-Status: Inappropriate [oe specific]

Signed-off-by: Hongxu Jia <hongxu.jia@windriver.com>

Update context for anaconda 32.

Signed-off-by: Kai Kang <kai.kang@windriver.com>
---
 pyanaconda/payload/livepayload.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/pyanaconda/payload/livepayload.py b/pyanaconda/payload/livepayload.py
index cca38c0510..dbcb04fee0 100644
--- a/pyanaconda/payload/livepayload.py
+++ b/pyanaconda/payload/livepayload.py
@@ -82,7 +82,7 @@ class LiveImagePayload(Payload):
                                     (self.data.method.partition,))
             if errorHandler.cb(exn) == ERROR_RAISE:
                 raise exn
-        rc = payload_utils.mount(osimg_path, INSTALL_TREE, fstype="auto", options="ro")
+        rc = payload_utils.mount(osimg_path, INSTALL_TREE, fstype="auto", options="ro,noload")
         if rc != 0:
             raise PayloadInstallError("Failed to mount the install tree")
 
-- 
2.7.4
